float32 pos_x
float32 pos_y
float32 pos_z
float32 q1
float32 q2
float32 q3
float32 q4
float32 gripper